packages/story-editor/src/components/panels/design/layer/layer.js

```diff
@@ -364,8 +363,13 @@ function Layer({ element }) {
   });
 
   const layerId = `layer-${element.id}`;
+  const isNested = element.groupId;
 
-  const lockTitle = element.isLocked
+  const lockTitle = isNested
+    ? element.isLocked
+      ? __('Group is Locked', 'web-stories')
+      : __('Group is Unlocked', 'web-stories')
```

Comment:
This seems a bit misleading - there's a locked icon, but the tooltip says "Group is Unlocked":

<img width="146" alt="Screen Shot 2022-06-13 at 12 04 49" src="https://user-images.githubusercontent.com/637548/173330662-9ba015c1-3d1a-40e0-94f9-4f0852646691.png">

Maybe these layers inside a group should use the regular lock/unlock icons, but just have them at .5 opacity to indicate they cannot be individually toggled instead of using this filled lock icon?